junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvArgumentsProvider.java

```diff
@@ -47,26 +50,40 @@ public void accept(CsvSource annotation) {
 		settings.getFormat().setQuoteEscape('\'');
 		settings.setEmptyValue(this.annotation.emptyValue());
 		settings.setAutoConfigurationEnabled(false);
+
 		CsvParser csvParser = new CsvParser(settings);
 		AtomicLong index = new AtomicLong(0);
 
+		final DefaultConversionProcessor processor = getConversionProcessor();
```

Comment:
- Remove `final` modifier.
- Why are did use a `DefaultConversionProcessor` as the type of the variable instead of `ConversionProcessor`?

Reply:
The `final` modifier is gone.

I can't use the `ConversionProcessor` because the `applyConversions` Method is not part of this Interface. Therefore the `DefaultConversionProcessor` is the most common class that I could have used.

I also thought that `ConversionProcessor` should define `applyConversions`. However this is mainly because of the name of this interface. The documentation of the class is pretty specific about what its intend is:

> A basic interface for classes that associate {@link Conversion} implementations with fields of a given input/output.

Reply:
Understood.